packages/fiori/src/types/TimelineLayout.js

```diff
@@ -0,0 +1,40 @@
+import DataType from "@ui5/webcomponents-base/dist/types/DataType.js";
+
+/**
+ * @lends sap.ui.webcomponents.fiori.types.TimelineLayout.prototype
+ * @public
+ */
+const AxistOrintation = {
```

Comment:
typo -> Axist

Reply:
And, we usually use plural form for the object of the type and using the same name as the "type"
(see other types in the framework).

So, if the file and type is called TimelineLayout this should become -> const TimelineLayout**s**

Reply:
typo -> Done

Reply:
Done